cli/src/main/resources/help/shutdown.adoc

```diff
@@ -16,13 +15,29 @@ SYNOPSIS
 *shutdown cluster*
 
 
+USAGE CONSIDERATIONS
+--------------------
+*shutdown server*
+
+To prevent data loss, you should wait for any rebalancing operations to complete before stopping server instances.
+
+*shutdown cluster*
```

Comment:
suggestion: add another new line before this so you have an empty line in `help shutdown`